src/lib/ActionPrototype.js

```diff
@@ -1,29 +1,57 @@
-var fetcher = require('../lib/jsonFetcher');
-var AppDispatcher = require('../lib/ep-dispatcher');
-var KeyMirror = require('keymirror');
+import fetcher from '../lib/jsonFetcher'
+import AppDispatcher  from '../lib/ep-dispatcher'
```

Comment:
yeah, good point. I brought it over, just wholesale from the ep-code base. We can probably just call it Dispatcher. When we finally use it as a dependency, let's change it.